sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/IndentCheck.java

```diff
@@ -43,10 +43,18 @@
 
   private static final String MESSAGE = "Make this line start at column %s.";
 
-  @RuleProperty(key = "indentSize", defaultValue = "2")
+  @RuleProperty(
+    key = "indentSize",
+    description = "Number of white-spaces of an indent. If this property is not set, we just check that the code is indented.",
+    type = "INTEGER",
+    defaultValue = "2")
   private int indentSize = 2;
 
-  @RuleProperty(key = "tabSize", defaultValue = "2")
+  @RuleProperty(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder annotation arguments to match the order of declaration. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS3340)